### PR TITLE
package: Add support for AlmaLinux 9

### PR DESCRIPTION
**Pull request by HashidaTKS**
Add support for AlmaLinux 9

* [x] Fix package.yml
* [x] Add a new Dockerfile
* [x] Pass CI

### CODE REVIEW

**Comment by daipom** [comment]
Your Dockerfile seems to install libraries correctly!
However, the following error occurs.

https://github.com/HashidaTKS/milter-manager/runs/7314812623?check_suite_focus=true#step:7:120

>  Error: No matching repo to modify: powertools.

Do you have any idea of this error?

**Comment by daipom** [comment]
Hmm, 
Maybe `$ dnf config-manager --set-enabled powertools` fails and throwing this error.

**Comment by HashidaTKS** [comment]
under investigation...

**Comment by HashidaTKS** [comment]
> dnf config-manager --set-enabled powertools

AlmaLinux 9 don't have powertools, we should use `crb` instead.
(Thanks, @kou )

**Comment by HashidaTKS** [comment]
Memo:

When I remove `scl-utils` from `dnf install`, I got the following error.

https://github.com/HashidaTKS/milter-manager/runs/7315891306?check_suite_focus=true#step:7:1089

```
/host/build.sh: line 25: scl: command not found
/host/build.sh: line 25: scl: command not found
```

When I add `scl-utils` to `dnf install`, I got the following error.

https://github.com/HashidaTKS/milter-manager/runs/7315891306?check_suite_focus=true#step:7:1090

```
Unable to get file status /etc/scl/conf/rh-ruby30: No such file or directory
Unable to get file status /etc/scl/conf/rh-ruby30: No such file or directory
```

File: package/yum/almalinux-9/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+ARG FROM=almalinux:9
+FROM ${FROM}
+
+ARG DEBUG
+
+ENV SCL=rh-ruby30
```
Comment (kou) [inline package/yum/almalinux-9/Dockerfile:6]: We need to remove this.

File: package/yum/almalinux-9/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    glib2-devel \
+    gtk-doc \
+    intltool \
+    make \
+    pkgconfig\(gobject-introspection-1.0\) \
+    ruby \
+    ruby-devel \
+    rubygem-rexml \
+    rpmdevtools \
+    scl-utils \
```
Comment (kou) [inline package/yum/almalinux-9/Dockerfile:25]: This is needless.

File: package/yum/almalinux-9/Dockerfile (hunk 1 above)
Comment (kou) [reply package/yum/almalinux-9/Dockerfile:6]: If this is defined, SCL is used.

File: .github/workflows/package.yml [hunk 3]
```diff
@@ -69,17 +69,23 @@ jobs:
...
           - label: CentOS 7
             id: centos-7
             task-namespace: yum
             target: centos-7
             test-docker-image: centos:7
-          - label: Debian GNU/Linux bullseye
-            id: debian-bullseye
-            task-namespace: apt
-            target: debian-bullseye
-            test-docker-image: debian:bullseye
```
Comment (kou) [inline .github/workflows/package.yml:82]: Why is this block removed? (To reduce CI time?)

Comment (HashidaTKS) [reply .github/workflows/package.yml:82]: Ah, It's just a mistake. Will revert it.

File: package/yum/almalinux-9/Dockerfile (hunk 1 above)
Comment (HashidaTKS) [reply package/yum/almalinux-9/Dockerfile]: Thanks! I am trying it!

**Comment by HashidaTKS** [comment]
We need also change these lines.

https://github.com/milter-manager/milter-manager/blob/master/configure.ac#L259-L290

**Comment by HashidaTKS** [comment]
The action for this failed with a following reason.
I am investigating about it...

> chmod: cannot access '/root/rpmbuild/BUILDROOT/milter-manager-2.1.6-1.el9.x86_64/etc/cron.d/milter-manager-log': No such file or directory

https://github.com/HashidaTKS/milter-manager/runs/7401034900?check_suite_focus=true#step:7:16816

**Comment by HashidaTKS** [comment]
It seems that a command `/usr/bin/install -c -m 644 milter-manager-log '/root/rpmbuild/BUILDROOT/milter-manager-2.1.6-1.el9.x86_64/etc/cron.d'` was not executed in AlmaLinux 9.
It was executed in AlmaLinux 8 (which succeeded to build).
 

https://github.com/HashidaTKS/milter-manager/runs/7401034864?check_suite_focus=true#step:7:3850

**Comment by HashidaTKS** [comment]
I think we should add `redhat9` into `REDHAT_PLATFORM`.

**Comment by daipom** [comment]
Indeed, why wasn't there a version 9 there...
Sorry for the trouble.

**Comment by HashidaTKS** [comment]
https://github.com/HashidaTKS/milter-manager/runs/7403834076?check_suite_focus=true

The phase "Build with docker" passed.

**Comment by HashidaTKS** [comment]
@kou 

Would you please re-review this?

**Comment by kou** [comment]
Thanks!

**Pull request merged**